src/rpc/blockchain.cpp

```diff
@@ -1479,6 +1479,9 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
             "{\n"
             "  \"time\": xxxxx,        (numeric) The timestamp for the statistics in UNIX format.\n"
             "  \"txcount\": xxxxx,     (numeric) The total number of transactions in the chain up to that point.\n"
+            "  \"block_count\": xxxxx, (numeric) Size of the window in number of blocks.\n"
+            "  \"txratecount\": xxxxx, (numeric) The number of transactions in the window.\n"
+            "  \"txrateinterval\": xxxxx, (numeric) The elapsed time in the window.\n"
```

Comment:
I agree to rename those names and to add condition to the help text.
But I wonder `txcount`/`txrate` should rename to `tx_count`/`tx_rate` or remain.
If the name of txcount is remained , I like `window_txcount`  than `window_tx_count`.
Which do you think is better?